tests/test_buttons.py

```diff
@@ -25,6 +53,10 @@ def test_button_enter_event(mocker, m5stickv):
     # Assert that the event flag is reset
     assert not button_enter.event()
 
+    # Simulate button release
```

Comment:
This is not simulating a release, it's simulating the device has no button Enter, thus `value()` returns default (released) value: 1